src/vtol.py

```diff
@@ -155,9 +127,10 @@ def go_to(self, point):
                 break
         print("Target reached")
 
+
     def land(self):
         '''Commands vehicle to land'''
-        self.mode = VehicleMode("LAND")
+        self.mode = VehicleMode(self.land_mode)
```

Comment:
I like that this was moved to the init method

Reply:
Yeah, had to abstract as quadplane uses "QLAND" and quad uses "LAND", didnt want to rewrite code